src/layout/section/cardsGridSection/CardsGridSection.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import cc from 'classcat'
+
+import { Button } from '../../../button'
+import { SpacingDivider, SpacingDividerSize } from '../../../divider/spacingDivider'
+import { BaseSection, SectionContentSize } from '../../../layout/section/baseSection'
+import { TripCardProps } from '../../../tripCard'
+import { TextDisplay1 } from '../../../typography/display1'
+import { StyledCardsGridSection } from './CardsGridSection.style'
+
+type ChildrenProps = React.ReactElement<TripCardProps>[] | React.ReactElement<TripCardProps>
+
+export type CardsGridSectionProps = Readonly<{
+  children: ChildrenProps
+  className?: string
+  contentSize?: SectionContentSize
+  title?: string | JSX.Element
+  buttonTitle?: string
+  buttonHref?: string | JSX.Element
+}>
+
+export const CardsGridSection = ({
+  children,
+  className = '',
+  contentSize = SectionContentSize.SMALL,
+  title,
+  buttonTitle,
+  buttonHref,
+}: CardsGridSectionProps) => (
+  <StyledCardsGridSection
+    className={cc([
+      'kirk-cardsGridSection-wrapper',
+      {
+        'kirk-cardsGridSection-wrapper--large': contentSize === SectionContentSize.LARGE,
+      },
+      className,
+    ])}
+  >
+    {title && (
+      <BaseSection contentSize={contentSize}>
+        <h2 className="kirk-cardsGridSection-title">
```

Comment:
You can convert all this on a styled-component

```suggestion
      const Title =   styled(TextDisplay1)
```

Reply:
I will remove the className and use `styled(TextDisplay1)` instead but I can't remove the `h2`.